docs/microsoft-365-developer-program-faq.yml

```diff
@@ -43,9 +43,11 @@ sections:
           Note that if you delete your Developer Program account, you can only get a subscription after you have had your existing subscription for more than 60 days.
 
       - question: |
-          How can I get a Microsoft 365 developer subscription via the Developer Program?  
+          How can I get a Microsoft 365 E5 developer subscription via the Developer Program?  
         answer: |
-          When you join the Microsoft 365 Developer Program, you qualify to get a free Microsoft 365 E5 developer subscription. This is a special subscription designed to be used for application development only. For more information, see [Set up a Microsoft 365 developer subscription](microsoft-365-developer-program-get-started.md).
+          For qualified members, you're invited to set up a Microsoft 365 E5 developer subscription after you join the Microsoft 365 Developer Program. If you don't immediately qualify for a developer subscription, after you join the program, you'll have the option to provide additional information to validate that you're a qualified member. Submit the information and if you qualify, you'll receive an email inviting you to set up your subscription within two weeks. If you don't receive an email, you don't currently qualify for a subscription.
```

Comment:
Qualified members are invited to set up a Microsoft 365 E5 developer subscription after joining the Microsoft 365 Developer Program.  If you don't immediately qualify, after you join the program, you will have the option to provide additional information to validate your qualification. Once you submit the required details, if you qualify, you will receive an email inviting you to set up your subscription within two weeks. If you do not receive an email, that means you do not currently qualify for a subscription.